btr-web/btr-main-app/pnpm-lock.yaml

```diff
@@ -7,18 +7,21 @@ settings:
 dependencies:
   pinia:
     specifier: ^2.1.7
-    version: 2.1.7(vue@3.4.21)
+    version: 2.1.7(vue@3.4.26)
+  uuid:
+    specifier: ^9.0.1
+    version: 9.0.1
```

Comment:
I think this is the only thing that needed to change in the package-lock? In the future can you put other depencency upgrades in a separate PR? Especially for PRs like this its hard to tell if anything isn't working is due to dependency changes